src/WebJobs.Script/Description/ProxyMetadata.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    public class ProxyMetadata : FunctionMetadata
```

Comment:
>ProxyMetadata [](start = 17, length = 13)

Nobody else derives from FunctionMetadata. 
Can we avoid the new base class and instead have a FunctionMetadata instance with the right shape?  could it literally be a http trigger (but with SCriptType set to Gilbralter). Method/UriTemplate  properties here could then be moved to where we'd expect them on HttpTrigger.